### ISSUE TITLE
Sally's changes to Figure 1 (sc2ts schematic)

### ISSUE THREAD

**Issue by hyanwong** [jeromekelleher/sc2ts-paper#156]
Sally made 3 comments on @szhan's sc2ts schematic (which used to be figure 7, now moved to fig 1)

1. The node distinctions are not consistent with the subgraph plots (e.g. Fig 6). For example, we could have non-sample nodes as smaller, and recombination nodes as black (not blue). I'm not sure we *need* to make them consistent (and @jeromekelleher agrees), but I guess if it is easy to do so, it wouldn't hurt.
2. In figures D and E, the "newly added sample" nodes should presumably be at a different time (the most recent time), and none of the previously added sample nodes should be at the bottom of the tree
3. Figure D is logically problematic, because the middle node should have been added to the left hand node.

Along with point 3., there is a problem because we need to indicate that the two mutations are actually *the same* mutation. We need some way to indicate that. One possibility is to use a different symbol?

**Comment by hyanwong** [comment]
After discussion, for point 3, rather than using a different symbol, perhaps a label on subfigures C and D pointing to "identical mutations" would be easiest?

**Comment by jeromekelleher** [comment]
@szhan - do you think we'll get these in today?

**Comment by szhan** [comment]
WIP.

1. I have updated the Legend. I've added the symbols for recombination nodes and inserted nodes that Yan uses in the other figures.
2. In panels D and E, the older sample nodes are moved up, situated higher than the newly added sample nodes.

The positions of the nodes in panels D and E needs some tweaking. Also, I still have to add labels next to the nodes.

<img width="451" alt="Screen Shot 2023-06-02 at 1 43 34 PM" src="https://github.com/jeromekelleher/sc2ts-paper/assets/5580375/7db197d2-73fc-42fb-a523-1df290cf3125">

**Comment by szhan** [comment]
I also removed the texts "Mutation-collapsing node" and "Reversion-pushed node" in panels D and E. I think that it is clear enough without them.

**Comment by hyanwong** [comment]
Thanks Shing. I reckon you can probably move the older sample nodes downwards a fraction more - no need for them to be halfway between the newly added samples and the root?

**Comment by szhan** [comment]
I think these labels make it more clear?

<img width="875" alt="Screen Shot 2023-06-02 at 2 04 24 PM" src="https://github.com/jeromekelleher/sc2ts-paper/assets/5580375/e7976290-08e9-4a5c-ad4a-23362349bb31">

**Comment by hyanwong** [comment]
They do help, yes. But I wonder if it might still help to indicate somehow that the two mutations are the same? Maybe it's a bit clumsy though?

<img width="715" alt="Screenshot 2023-06-02 at 14 14 06" src="https://github.com/jeromekelleher/sc2ts-paper/assets/4699014/66cfb291-e9f1-4a2f-ace2-4edef4f345cd">

**Comment by szhan** [comment]
Or just highlight them?
<img width="559" alt="Screen Shot 2023-06-02 at 2 17 08 PM" src="https://github.com/jeromekelleher/sc2ts-paper/assets/5580375/25f077c3-83bd-48bd-9d7d-2019584ff8ac">

**Comment by szhan** [comment]
This is WIP of panel E.

<img width="679" alt="Screen Shot 2023-06-02 at 2 13 51 PM" src="https://github.com/jeromekelleher/sc2ts-paper/assets/5580375/0a713a6d-4c79-49ac-8309-e974e8c54778">

**Comment by szhan** [comment]
I still somewhat prefer having the middle sample node half way.

**Comment by szhan** [comment]
<img width="420" alt="Screen Shot 2023-06-02 at 2 31 33 PM" src="https://github.com/jeromekelleher/sc2ts-paper/assets/5580375/df249860-a20d-4b9e-b564-29cf686cc403">

**Comment by szhan** [comment]
Maybe the text highlighting in panel D is not obvious enough. Not when the full figure is displayed and panel D is not zoomed in.

**Comment by hyanwong** [comment]
I think the mutation labels are quite small to read - they might be overlooked.

**Comment by szhan** [comment]
I don't see harm in using more colours, since we have only used three non-greyscale colours so far.

<img width="1056" alt="Screen Shot 2023-06-02 at 2 58 20 PM" src="https://github.com/jeromekelleher/sc2ts-paper/assets/5580375/be505099-ff3a-4dc6-aba5-6183faa27d11">

**Comment by hyanwong** [comment]
Good point. I think colours on the mutation bars themselves would be confusing, but colours on the labels wrks well, I think.

**Comment by hyanwong** [comment]
Can you make the label font about 10% bigger, or does that cause problems?

**Comment by szhan** [comment]
Actually, I think I'mma change the colours for all the symbols. We could, for example, make the text labels in panel C to pop by using red. Then, use a different colour for mutations, say, blue. Parent 1 could have a different colour, say, orange.

**Comment by hyanwong** [comment]
Hmm, I'de be careful. Too many colours can be confusing. It's good to use them to highlight very specific things.

**Comment by hyanwong** [comment]
Personally, I think what you have is perfect, colour-wise (maybe slightly darker blue for A100G

**Comment by szhan** [comment]
How about this?

![Uploading Screen Shot 2023-06-02 at 3.25.30 PM.png…]()

**Comment by jeromekelleher** [comment]
LGTM

**Comment by szhan** [comment]
Checking once more and then PR.

**Comment by szhan** [comment]
Missed that one mutation in on the right local tree. Darn.

<img width="444" alt="Screen Shot 2023-06-02 at 3 56 11 PM" src="https://github.com/jeromekelleher/sc2ts-paper/assets/5580375/2348439d-c51e-4e78-9d3b-4ab16b25bae0">

**Comment by hyanwong** [comment]
Mentioned to Shing on Slack - also missing labels to show mutations are identical in subfig C

**Comment by szhan** [comment]
<img width="729" alt="Screen Shot 2023-06-02 at 6 49 56 PM" src="https://github.com/jeromekelleher/sc2ts-paper/assets/5580375/01f78995-a771-4ff0-9002-fecc0c3eb474">

**Comment by szhan** [comment]
Which set of colours is better?

<img width="1031" alt="Screen Shot 2023-06-02 at 7 00 03 PM" src="https://github.com/jeromekelleher/sc2ts-paper/assets/5580375/ed65967f-60fc-489d-bb6e-559668bda46f">

**Comment by hyanwong** [comment]
I like the right hand (orange text).

I think, however, that we should be consistent and also have the same colour text in 1C.

**Comment by szhan** [comment]
<img width="1071" alt="Screen Shot 2023-06-02 at 7 05 04 PM" src="https://github.com/jeromekelleher/sc2ts-paper/assets/5580375/265fd74b-dbd0-468f-87b9-a36b42a42978">

**Comment by szhan** [comment]
Latest version

<img width="450" alt="Screen Shot 2023-06-02 at 7 13 18 PM" src="https://github.com/jeromekelleher/sc2ts-paper/assets/5580375/22a90788-6275-422f-afbc-df59873859e1">

**Comment by hyanwong** [comment]
Looks fine. Is there any way to avoid the dotted arrows in C. by placing the label twice, like it is in the panel below that (I realise there might not be room, though.

Also, I guess I could change the mutation labels in the subplots to the blue you are using for mutation bars? This is what that would look like:

![image](https://github.com/jeromekelleher/sc2ts-paper/assets/4699014/545d2d01-4b84-4ba0-b5fc-dd96f85f3076)

**Comment by jeromekelleher** [comment]
Are we happy with the latest version just merged?

**Comment by szhan** [comment]
I kind of like the blue mutation labels. Maybe let's go with that for now, since it is quickest?

**Comment by szhan** [comment]
Removed the dotted arrows.

<img width="465" alt="Screen Shot 2023-06-04 at 10 44 10 AM" src="https://github.com/jeromekelleher/sc2ts-paper/assets/5580375/a627d3e1-6423-4db6-b462-01c857eea768">

**Comment by hyanwong** [comment]
Look great to me. Do we want to add something to the legend for describe "G7A", e.g. instead of  "Mutation", we could say "Mutation (may be labelled e.g. G7A for inherited state G, position 7, derived state A)". Or a shorter label if you can think of one!

**Comment by szhan** [comment]
I see labels like "G7A" often in the literature, so maybe our target audience is comfortable with it.

Also, in the legends, recombination node, inserted node, and sample nodes are listed in different orders. Should we sync them too? If so, maybe stick with that order?

Maybe I should lowercase the text in the legend as well.

**Comment by hyanwong** [comment]
> I see labels like "G7A" often in the literature, so maybe our target audience is comfortable with it.

Yes, or we could commend in the caption instead. A shorter legend description would be "Mutation (may be labelled with state change and position)"

> Also, in the legends, recombination node, inserted node, and sample nodes are listed in different orders. Should we sync them too? If so, maybe stick with that order?

Good point. I don't think it's important, but I guess sample nodes are the first ones encountered, which is why I put them first. But we could go for alphabetical order:

inserted
recombination <- this is a type of inserted node, so might be useful to put after the inserted node
sample node
sample node (newly added)

> Maybe I should lowercase the text in the legend as well.

I'm happy with either. Which looks best to you?

**Comment by hyanwong** [comment]
I think close and we can open up a different issue for the minor tweaks discussed above, as they are very minor indeed.
